kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -140,7 +140,7 @@ public void testGetSolverState() throws Exception {
         assertEquals( SOLVER_1_ID, returnedInstance.getSolverId() );
         assertEquals( SolverInstance.getSolverInstanceKey( CONTAINER_1_ID, SOLVER_1_ID ), returnedInstance.getSolverInstanceKey());
         assertEquals( SolverInstance.SolverStatus.NOT_SOLVING, returnedInstance.getStatus());
-        assertNull( returnedInstance.getScore() );
+        assertNull( returnedInstance.getScoreWrapper() );
```

Comment:
@ge0ffrey Makes more sense now seeing it from a different angle (not just from XML perspective). The score wrapper is now always assigned, no matter if the score is null or not.
For this purpose I removed the null check from `ScoreWrapper` constructor, as it's valid to construct an instance which encapsulates null score.